src/features/comments.ts

```diff
@@ -0,0 +1,103 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import { PayloadAction } from '@reduxjs/toolkit';
+import { Comment } from '../types/Comment';
+import { getPostComments, createComment, deleteComment } from '../api/comments';
+
+interface CommentsState {
+  items: Comment[];
+  loaded: boolean;
+  hasError: boolean;
+}
+
+const initialState: CommentsState = {
+  items: [] as Comment[],
+  loaded: false,
+  hasError: false,
+};
+
+// Thunks
+export const fetchComments = createAsyncThunk(
+  'comments/fetchComments',
+  async (postId: number | undefined) => {
+    if (postId === undefined) {
+      return;
```

Comment:
Returning `undefined` when `postId` is not provided in the `fetchComments` thunk might lead to unexpected behavior. Consider handling this case more explicitly, perhaps by throwing an error or returning an empty array.